### PR TITLE
fix(release): bump all dependencies to latest

### PR DESCRIPTION
**Pull request by Pilpin**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[x] Other... Please describe: update dependencies
```

## What is the current behavior?

Dependencies are quite old

Issue Number: N/A

## What is the new behavior?
Dependencies are up-to-date.

## Does this PR introduce a breaking change?
I don't know
```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by NetanelBasal** [comment]
Did you did it using the ng update command?

**Comment by Pilpin** [comment]
> Did you did it using the ng update command?

I did not, but I just did it again, and it only replaced browerTarget to buildTarget in angular.json.
Should I do another PR to replace this one ?

**Comment by NetanelBasal** [comment]
We also need to:
- convert to standalone
- remove the module
- run the control flow migration
- convert the playground to standalone app

**Comment by Pilpin** [comment]
> We also need to:
> 
> * convert to standalone
> * remove the module
> * run the control flow migration
> * convert the playground to standalone app

Sure, I'll try and do it this week.

File: README.md [hunk 1]
```diff
@@ -58,6 +60,17 @@ import { HotkeysModule } from '@ngneat/hotkeys';
...
 ```
 
+### Standalone
+Add `HotkeysService` in your `ApplicationConfig` providers:
+
+```ts
+import {HotkeysService} from "@ngneat/hotkeys";
+
+bootstrapApplication(AppComponent, {
+  providers: [HotkeysService]
```
Comment (NetanelBasal) [inline README.md:70]: why we need it here? it's a root service

File: src/main.ts [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
+import { HotkeysService } from '@ngneat/hotkeys';
 
 if (environment.production) {
   enableProdMode();
 }
 
-platformBrowserDynamic().bootstrapModule(AppModule)
-  .catch(err => console.error(err));
+bootstrapApplication(AppComponent, {
+  providers: [importProvidersFrom(BrowserModule, BrowserAnimationsModule, NgbModalModule), HotkeysService],
```
Comment (NetanelBasal) [inline src/main.ts:15]: same question

File: projects/ngneat/hotkeys/src/lib/hotkeys.directive.ts [hunk 3]
```diff
@@ -13,59 +14,45 @@ interface Options {
...
-      .subscribe(e => this.hotkey.next(e));
+  hotkeys = input<string>();
+  isSequence = input(false);
+  hotkeysGroup = input<string>();
+  hotkeysOptions = input<Partial<Options>>({});
+  hotkeysDescription = input<string>();
+  @Output() hotkey = new EventEmitter<KeyboardEvent | Hotkey>();
+
+  constructor() {
+    effect(() => {
```
Comment (NetanelBasal) [inline projects/ngneat/hotkeys/src/lib/hotkeys.directive.ts:34]: Why do we need this change? ngOnChanges still works with signal inputs

Comment (netanel-utila) [reply projects/ngneat/hotkeys/src/lib/hotkeys.directive.ts:34]: I am not familiar with such best practice. effect is currently in dev preview

Comment (netanel-utila) [reply projects/ngneat/hotkeys/src/lib/hotkeys.directive.ts:34]: No. They are more stable than effect that its implementation was changed several times.

Comment (Pilpin) [reply projects/ngneat/hotkeys/src/lib/hotkeys.directive.ts:34]: Alright, I'll revert to ngchanges when I have time this week.


> No. They are more stable than effect that its implementation was changed several times.

@netanel-utila could you let me know how you came to this conclusion ? Is there an ”easy” way to know how confident the Angular devs are with the developer preview features ?

Comment (NetanelBasal) [reply projects/ngneat/hotkeys/src/lib/hotkeys.directive.ts:34]: experience

**Comment by Pilpin** [comment]
[5323d0c](https://github.com/ngneat/hotkeys/pull/91/commits/5323d0cbc38279a2a502f794a3c278a32d80adbb) I also bumped the version number from 2.0.0 to 3.0.0

File: projects/ngneat/hotkeys/src/lib/hotkeys.directive.ts [hunk 4]
```diff
@@ -1,71 +1,71 @@
...
+      coercedHotkeys.map((hotkey) => {
+        return this.isSequence()
           ? this.hotkeysService.addSequenceShortcut({ ...hotkey, element: this.elementRef.nativeElement })
           : this.hotkeysService.addShortcut({ ...hotkey, element: this.elementRef.nativeElement });
-      })
+      }),
     )
-      .pipe(mergeAll())
-      .subscribe(e => this.hotkey.next(e));
+      .pipe(takeUntilDestroyed(this.destroyRef), mergeAll())
```
Comment (NetanelBasal) [inline projects/ngneat/hotkeys/src/lib/hotkeys.directive.ts:68]: why you need this change? you already have the subscription and the unsubscribe call

**Comment by Pilpin** [comment]
Hello, is there something else you would like me to change ?

**Pull request merged**